tests/datatypes/test_bytes.py

```diff
@@ -43,6 +43,20 @@ def test_iter(self):
             print([b for b in b'hello world'])
         """)
 
+    def test_find(self):
+        self.assertCodeExecution("""
+            print(b''.find(b'a'))
+            print(b'abcd'.find(b''))
+            print(b'abcd'.find(b'...'))
+            print(b'abcd'.find(b'ab'))
+            print(b'abcd'.find(b'bc'))
+            print(b'abcd'.find(b'cd'))
+            print(b'abcd'.find(b'cd', 2))
+            print(b'abcd'.find(b'ab', 3))
+            print(b'abcd'.find(b'cd', 2, 3))
+            print(b'abcd'.find(b'ab', 3, 4))
```

Comment:
@eliasdorneles thanks, I have added use cases given by you and have fixed the bug.